[2024-08-12T21:21:57.229+0100] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-12T21:21:57.257+0100] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: jobs_etl.indeed_scraper scheduled__2024-08-11T18:00:00+00:00 [queued]>
[2024-08-12T21:21:57.263+0100] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: jobs_etl.indeed_scraper scheduled__2024-08-11T18:00:00+00:00 [queued]>
[2024-08-12T21:21:57.263+0100] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-12T21:21:57.271+0100] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): indeed_scraper> on 2024-08-11 18:00:00+00:00
[2024-08-12T21:21:57.278+0100] {standard_task_runner.py:63} INFO - Started process 984552 to run task
[2024-08-12T21:21:57.281+0100] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'jobs_etl', 'indeed_scraper', 'scheduled__2024-08-11T18:00:00+00:00', '--job-id', '351', '--raw', '--subdir', 'DAGS_FOLDER/jobs_etl.py', '--cfg-path', '/tmp/tmp7x6ggynh']
[2024-08-12T21:21:57.282+0100] {standard_task_runner.py:91} INFO - Job 351: Subtask indeed_scraper
[2024-08-12T21:21:57.337+0100] {task_command.py:426} INFO - Running <TaskInstance: jobs_etl.indeed_scraper scheduled__2024-08-11T18:00:00+00:00 [running]> on host huser
[2024-08-12T21:21:57.428+0100] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='hassounibarka@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='jobs_etl' AIRFLOW_CTX_TASK_ID='indeed_scraper' AIRFLOW_CTX_EXECUTION_DATE='2024-08-11T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-11T18:00:00+00:00'
[2024-08-12T21:21:57.428+0100] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-12T21:22:28.182+0100] {logging_mixin.py:188} INFO - 10 Jobs found for data Scientist
[2024-08-12T21:22:53.757+0100] {logging_mixin.py:188} INFO - 17 Jobs found for data analyst
[2024-08-12T21:23:18.282+0100] {local_task_job_runner.py:310} WARNING - State of this instance has been externally set to restarting. Terminating instance.
[2024-08-12T21:23:18.284+0100] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-08-12T21:23:18.299+0100] {process_utils.py:132} INFO - Sending 15 to group 984552. PIDs of all processes in the group: [984878, 984882, 984883, 984893, 984894, 984922, 984896, 984942, 984967, 985006, 984920, 984552]
[2024-08-12T21:23:18.300+0100] {process_utils.py:87} INFO - Sending the signal 15 to group 984552
[2024-08-12T21:23:18.301+0100] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-08-12T21:23:18.302+0100] {logging_mixin.py:188} INFO - Error URL
[2024-08-12T21:23:18.302+0100] {logging_mixin.py:188} INFO - 0 Jobs found for data engineer
[2024-08-12T21:23:18.315+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=985006, status='terminated', started='21:22:55') (985006) terminated with exit code None
[2024-08-12T21:23:18.335+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984920, status='terminated', started='21:22:54') (984920) terminated with exit code None
[2024-08-12T21:23:18.394+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984894, status='terminated', started='21:22:54') (984894) terminated with exit code None
[2024-08-12T21:23:18.395+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984922, status='terminated', started='21:22:54') (984922) terminated with exit code None
[2024-08-12T21:23:18.395+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984882, status='terminated', started='21:22:53') (984882) terminated with exit code None
[2024-08-12T21:23:18.395+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984896, status='terminated', started='21:22:54') (984896) terminated with exit code None
[2024-08-12T21:23:18.396+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984967, status='terminated', started='21:22:54') (984967) terminated with exit code None
[2024-08-12T21:23:18.396+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984883, status='terminated', started='21:22:53') (984883) terminated with exit code None
[2024-08-12T21:23:18.493+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984893, status='terminated', started='21:22:54') (984893) terminated with exit code None
[2024-08-12T21:23:18.494+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984942, status='terminated', started='21:22:54') (984942) terminated with exit code None
[2024-08-12T21:23:19.111+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984878, status='terminated', started='21:22:53') (984878) terminated with exit code None
[2024-08-12T21:23:43.818+0100] {logging_mixin.py:188} INFO - 30 Jobs found for web developer
[2024-08-12T21:24:13.574+0100] {logging_mixin.py:188} INFO - 13 Jobs found for mobile developer
[2024-08-12T21:24:18.312+0100] {process_utils.py:150} WARNING - process psutil.Process(pid=984552, name='airflow task runner: jobs_etl indeed_scraper scheduled__2024-08-11T18:00:00+00:00 351', status='sleeping', started='21:21:56') did not respond to SIGTERM. Trying SIGKILL
[2024-08-12T21:24:18.314+0100] {process_utils.py:87} INFO - Sending the signal 9 to group 984552
[2024-08-12T21:24:18.327+0100] {process_utils.py:80} INFO - Process psutil.Process(pid=984552, name='airflow task runner: jobs_etl indeed_scraper scheduled__2024-08-11T18:00:00+00:00 351', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='21:21:56') (984552) terminated with exit code -9
[2024-08-12T21:24:18.329+0100] {standard_task_runner.py:178} ERROR - Job 351 was killed before it finished (likely due to running out of memory)
